notebooks/openvino/phi-2_on_mtl.ipynb

```diff
@@ -0,0 +1,571 @@
+{
```

Comment:
also might make sense to rename the file to something like code_generation_weight_quantization.ipynb

Reply:
We prefer the notebook will stay with the current name, will that be a problem?

Reply:
`optimized_chatbot.ipynb` sounds good?

Reply:
@echarlaix how about optimized_llm_inference.ipynb?
we are ok with whatever you decide 
anyway - can you make the change and accept the PR before the weekend?
@ofirzaf

Reply:
works for me ! will merge it now to not block anything on your side